hydra-main/src/main/java/com/addthis/hydra/job/spawn/balancer/HostScore.java

```diff
@@ -16,11 +16,13 @@
 final class HostScore {
 
     private final double meanActiveTasks;
+    private final long diskFreeBytes;
     private final double usedDiskPercent;
     private final double overallScore;
 
-    HostScore(double meanActiveTasks, double usedDiskPercent, double overallScore) {
+    HostScore(double meanActiveTasks, double usedDiskPercent, long diskFreeBytes, double overallScore) {
         this.meanActiveTasks = meanActiveTasks;
+        this.diskFreeBytes = diskFreeBytes;
```

Comment:
This doesn't seem immediately useful to the purported purpose of calculating the host score. The only use I see appears to still have host states themselves in scope (or very near).

When dealing with spawn, I suggest thinking twice before adding a line of code, and thinking five more times before adding fields.

It isn't too late to add it later if it becomes part pf the actual score or external function of the class.